HEC.FDA.Model/metrics/SystemPerformanceResults.cs

```diff
@@ -18,10 +18,8 @@ public class SystemPerformanceResults : ValidationErrorLogger
         #region Fields
         private const string AEP_ASSURANCE_TYPE = "AEP";
         private const string STAGE_ASSURANCE_TYPE = "STAGE";
-        private const string AEP_ASSURANCE_FOR_PLOTTING = "AEP_PLOT";
```

Comment:
This seems impactful for plotting. We were plotting using a different histogram for some reason. Maybe that's why our plot was all stair steppy

Reply:
resolved in person